examples/README.md

```diff
@@ -7,3 +7,23 @@ In this folder there are examples taken from the delta.io quickstart guide and d
 ### Instructions
 * To run an example in Python run `spark-submit --packages io.delta:delta-core_2.12:{Delta Lake version} PATH/TO/EXAMPLE`
 * To run the Scala examples, `cd examples/scala` and run `./build/sbt "runMain example.{Example class name}"` e.g. `./build/sbt "runMain example.Quickstart"`
+
+### Docker Instructions
+
+#### Building the image
```

Comment:
Does not make sense to create thsi many single line sections. Please collapse them into "Running the docker image" and "Running examples"